src/Microsoft.ML.Data/TrainCatalog.cs

```diff
@@ -366,6 +368,99 @@ internal BinaryClassificationTrainers(BinaryClassificationCatalog catalog)
             return result.Select(x => new CrossValidationResult<CalibratedBinaryClassificationMetrics>(x.Model,
                 Evaluate(x.Scores, labelColumn), x.Scores, x.Fold)).ToArray();
         }
+
+        /// <summary>
+        /// The list of trainers for performing binary classification.
+        /// </summary>
+        public CalibratorsCatalog Calibrators { get; }
+
+        public sealed class CalibratorsCatalog : CatalogInstantiatorBase
```

Comment:
>CalibratorsCatalog [](start = 28, length = 18)

i need xml doc too.

Reply:
To be honest none of other `CatalogInstantiatorBase` inherited classes has one. And I don't plan to fix it right now. Documentation can wait till next month.

---
In reply to: [261052357](https://github.com/dotnet/machinelearning/pull/2766#discussion_r261052357) [](ancestors = 261052357)